ember-accounts/app/templates/settings/user-profile.hbs

```diff
@@ -14,5 +14,5 @@
 </div>
 
 <div class="question-label">
-  <strong>Have questions?</strong> Email us anytime at support@huboard.com
+  <strong>Have questions?</strong> Email us anytime at <a href='mailto:support@huboard.com'> support@huboard.com </a>
```

Comment:
Some browsers are dumb about leading/trailing whitespace and anchor tags. Trim?

:+1: for adding `mailto:`!